examples/nimble-typescript/README.md

```diff
@@ -0,0 +1,117 @@
+# Nimble Typescript Getting Started
+
+This example project creates a simple counter application using Vanilla (no framework) Typescript and the Nimble Design System.
+
+## Prerequisites
+
+1. Install VSCode  (or your editor of choice) - <https://code.visualstudio.com/>  
+
+    VSCode is a free editor (borderline IDE) that is very popular in web development.
+
+2. Install Node.js - <https://nodejs.org/en/download/>  
+
+    Node.js is an open-source, cross-platform, JavaScript runtime environment that executes JavaScript code outside of a web browser.
+
+## Start
+
+Start Example:
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/ni/nimble/tree/typescript-example/examples/nimble-typescript/start)
+
+Use the Stackblitz link to launch the starting point of the example. You can download the example as a zip file, extract it, and open it in your local editor.
+
+![Screenshot of Download project button in StackBlitz](download-project.png)
+
+Open the project folder in your editor and run `npm install` to install all the dependencies needed to run the project.
+
+Run `npm run dev` to run the development version of the application and see it in your web browser.
+
+## Integrating Nimble
+
+1. Add the nimble components to your project dependencies.  
+`npm install @ni/nimble-components`
+1. In `counter.ts` import the nimble-button reference and update the parameter type to `Button` for the `setupCounter` function.  
```

Comment:
I think this repo standardized on keeping the count up to date in markdown numbered lists so that it's more readable in raw text form.
```suggestion
2. In `counter.ts` import the nimble-button reference and update the parameter type to `Button` for the `setupCounter` function.  
```